frontend/frontend/protocols/ssh/_proxy_handler.py

```diff
@@ -5,7 +5,9 @@
 import logging
 import socket
 import threading
-from time import sleep, time
+from time import sleep
```

Comment:
```suggestion
from time import sleep, time
```

Add missing time import in proxy_handler